### PR TITLE
react-week1-homework

### PR DESCRIPTION
**Pull request by qamarfj**

### CODE REVIEW

File: react/week1/todo/src/ToDo.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+export const ToDo = ({ title, deadLine }) => {
+  return (
+    <li>
+      <h3> {title}</h3>
```
Comment (braedongough) [inline react/week1/todo/src/ToDo.js]:
```suggestion
      <h3>{title}</h3>
```

File: react/week1/todo/src/ToDo.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+export const ToDo = ({ title, deadLine }) => {
```
Comment (braedongough) [inline react/week1/todo/src/ToDo.js]:
```suggestion
export const Todo = ({ title, deadLine }) => {
```

I wonder consider renaming this to keep it consistent with your Todos component.

**Review by braedongough — APPROVED** [review]
Great work! No major feedback 😄

**Review by Ekamba — CHANGES_REQUESTED** [review]
Hi @qamarfj , like @braedongough said your work is good. But remember  when you are are rendering a list of items you have to specify a key. You can read more about it here: https://reactjs.org/docs/lists-and-keys.html

**Pull request merged**